go.mod

```diff
@@ -32,6 +32,8 @@ require (
 	k8s.io/apimachinery v0.26.0
 )
 
+replace gopkg.in/yaml.v3 => github.com/colega/go-yaml-yaml v0.0.0-20220720070545-aaba007ebc22
```

Comment:
What's go-yaml-yaml advantage？

Reply:
it supports `value.DecodeWithOptions`, I can pass yaml.DecodeOptions{KnownFields: true} to support the KnownFields feature.

Reply:
there is a PR containing the feather. but not be merged.